global.json

```diff
@@ -4,7 +4,7 @@
   },
   "sdk": {
     "allowPrerelease": false,
-    "version": "6.0.400",
+    "version": "6.0.100",
```

Comment:
Only the `6.0.1xx` SDK series is currently being updated in Ubuntu repositories (`6.0.109` as of [.NET 6.0.9](https://github.com/dotnet/core/blob/main/release-notes/6.0/6.0.9/6.0.9.md)), so I believe `6.0.100` is our lowest common denominator. The `6.0.3xx` and `6.0.4xx` series that are also current should be permitted with `"rollForward": "latestFeature"`

Reply:
> Only the `6.0.1xx` SDK series is currently being updated in Ubuntu repositories (`6.0.109` as of [.NET 6.0.9](https://github.com/dotnet/core/blob/main/release-notes/6.0/6.0.9/6.0.9.md))

What problem does this cause?

Reply:
No `dotnet` command will run in the repository if the installed SDK doesn't meet the version requirements specified in this file. I usually do Bicep work in WSL, so I was unable to run the tests. I think you can work around this by manually installing a different SDK version, but that would mean forgoing automatic updates.

Do you know if there was a specific feature we needed from `6.0.400`? It looks like this was changed in #8428, but all the tests are passing with the `6.0.109` SDK.

Reply:
If there's no reason for the upgrade to 6.0.4xx then I don't have a problem downgrading it back. You might want to double-check this with Marcin first.

In future I wouldn't want us to avoid upgrading to support particular environments if there's a good reason to - I'd recommend looking into containers (local devcontainer / Codespaces), or just manually installing dotnet if it comes up again.

Reply:
I upgraded it when trying to fix the issues I was running into with source generators but I'm not sure if it's strictly needed. Can you double check that the source generator works correctly with `dotnet build`, VS editor, and VS build after downgrading? (In VS editor behavior and build can fail differently and independently if generator versions aren't right.)

Reply:
I found a Microsoft-maintained alternative PPA for dotnet SDK feature updates and was able to upgrade. The Ubuntu official repositories are probably locked on `6.0.1xx` for the next 18 months due to their LTS policy, but that's also why `apt install nodejs` will install node 12 without a third-party package feed.